tests/validation/cattlevalidationtest/core/common_fixtures.py

```diff
@@ -1661,3 +1681,136 @@ def validate_internal_lb(super_client, lb_service, services,
         resp = response[0].strip("\n")
         logger.info("Actual wget Response" + str(resp))
         assert resp in (expected_lb_response)
+
+
+def create_env_with_2_svc_hostnetwork(
+        client, scale_svc, scale_consumed_svc, port, sshport,
+        isnetworkModeHost_svc=False,
+        isnetworkModeHost_consumed_svc=False):
+
+    launch_config_svc = {"imageUuid": SSH_IMAGE_UUID_HOSTNET}
+    launch_config_consumed_svc = {"imageUuid": WEB_IMAGE_UUID}
+
+    if isnetworkModeHost_svc:
+        launch_config_svc["networkMode"] = "host"
+        launch_config_svc["labels"] = {
+            "io.rancher.container.dns": "true",
```

Comment:
We can just segregate labels to the var and use it on linke 1696 and 1704 as the label set seems to be the same